pkg/controller/disruption/disruption.go

```diff
@@ -98,7 +98,7 @@ type ReconcileMachineDisruption struct {
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileMachineDisruption) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	glog.Infof("Reconciling MachineDisruption triggered by %s/%s\n", request.Namespace, request.Name)
+	glog.V(4).Infof("Reconciling MachineDisruption triggered by %s/%s\n", request.Namespace, request.Name)
```

Comment:
this does not seems related to the title of this commit

Reply:
I will move all logs changes under the different commit, but it relatively small change, so I thought it is ok